.github/workflows/examples-telink.yaml

```diff
@@ -102,18 +102,29 @@ jobs:
             - name: clean out build output
               run: rm -rf ./out
 
-            - name: Build example Telink (B91) Lighting App with RPC, Shell and Factory Data
+            - name: Build tools required for Factory Data
               run: |
                   ./scripts/checkout_submodules.py --allow-changing-global-git-config --shallow --platform linux
                   ./scripts/build/gn_gen.sh
                   ./scripts/run_in_build_env.sh "ninja -C ./out/$BUILD_TYPE chip-cert chip-tool spake2p"
+
+            - name: clean out build output (keep tools)
```

Comment:
have you considered building using build_examples.py?
In that case you have the options to `--copy-artifacts-to` and only have the relevant bin files copied and then clean the output dir without GLOBIGNORE or similar

Reply:
build_examples.py can't build all required tools.
Also GLOBIGNORE doesn't works under CI.
Pushed MV to separate folder solution, just waiting if this solution will work fine.